src/main.rs

```diff
@@ -180,11 +194,23 @@ pub async fn main() {
                 let max_rows = header.extrinsics_root.rows;
                 let max_cols = header.extrinsics_root.cols;
                 let commitment = header.extrinsics_root.commitment.clone();
-
+				
+				if max_cols<3{
+					log::info!("lower number of chunks");
+					std::process::exit(0);
```

Comment:
Instead of exiting LC, should we keep polling for new blocks for a couple of times before possibly connecting to a new full node? 

This is a wider issue on the LC - how should we proceed when LC encounters errors on full nodes which would trigger potential switching to other (honest or non-malfunctioning) full nodes? cc: @prabal-banerjee

Reply:
A particular block data is unavailable (due to no or wrong proof) should make the LC instantly switch to new peers. Even after multiple tries, if it fails to retrieve data, it should report it as a fatal error. WDYT? @sh3ll3x3c

Reply:
@prabal-banerjee Fully agree - lets go with a simple retry mechanism with a single full node + switch to a different node if not successful. If if fails a couple of times with other nodes too - fatal error.

Now, the next question would be - do we have a discovery protocol (Libp2p) in place for light clients to discover full nodes (not just other light clients)?